tests/app/user/test_rest.py

```diff
@@ -776,6 +777,30 @@ def test_send_support_email_no_live_service(client, sample_user, mocker):
     mocked.assert_called_once_with(data | {'tags': ['z_skip_opsgenie', 'z_skip_urgent_escalation']})
 
 
+def test_send_contact_request(client, sample_user, mocker):
```

Comment:
Could you add a test case when the user exists and is a member of a live service? `test_send_support_email_with_live_service` does it but not on this endpoint